README.md

```diff
@@ -60,7 +60,24 @@ the extremely long folder name.
 
 ## Publishing
 
-Publishing happens through the CI/CD pipeline. See [package.json](.travis.yml)
+Publishing happens through the CI/CD pipeline. See [package.json](.github/workflows/ci-cd.yml)
+Steps for publishing an @next release:
```

Comment:
This needs some sort of header
### Steps for publishing... 

Same for `@latest`